### DISCUSSION TITLE
Welcome to MeshCentral Discussions

### DISCUSSION THREAD

**Discussion by Ylianst** [Ylianst/MeshCentral#3599] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JSuenram** [comment]
Hello,
I am leader of a small IT Company in Germany and started contributing here while searching for a ScreenConnect replacement.

What was just a try to use MeshCentral the first days has come to a real investment from our whole company to get MeshCentral to the top in Remote-Desktop-Support. 

We really like it and use it every day with great success.... we really appreciate that it is free for all and will stay like this hopefully forever!

**Reply by Ylianst** [reply]
You raise a point many have wondered before. MeshCentral is a community driven project and the intention is to keep it free and open source. MeshCentral makes sense as free software because it allows people to make sure of some technologies like Intel AMT.

**Reply by DaanSelen** [reply]
Hello @JSuenram how is MeshCentral 2 years into the future?

**Comment by jsastriawan** [comment]
Hello,

My name is Joko Sastriawan. I am working at the same company as Ylian. Mostly working on Intel AMT enabling. I will do my best to help around. :)

**Comment by Ylianst** [comment]
I should do the same! Hi everyone. I am Ylian Saint-Hilaire, a Principal Engineer at Intel and work on MeshCentral and MeshCommander full time. I love the project. I have been at Intel for 23 years now on the west coast of the US. I originally come from the French speaking part of Montreal and so, do speak French.

I love to code and work with this community. I can't possibly do everything, but I try to do as much as possible.

**Comment by krayon007** [comment]
I am Bryan Roe, a Staff Engineer at Intel. I work on various Mesh related tasks full time, and love the project as well. I've been at Intel for 22 years. I've always been in love with Software ever since my dad refused to buy me a game console when I was a kid, and instead bought me a computer and said I could play any game I wanted, as long as I wrote it myself... 

After reading various developer manuals cover to cover, I got my start by writing some software tools when I was 9 years old, that were integrated into my dad's company's purchasing and inventory tracking system. That's when I knew I wanted to make a career out of computer software.

**Comment by VINISHVV** [comment]
Hi,
I am Vinish Vijayan,handling the Technical support team of one of the indias No. 1 Financial institution. Earlier we are using paid tools like Team viewer / logmein for remote support. but now we are completely using meshcentral.

 Meshcentral realy helps me in many ways as we compare to paid tools.also it reduces 50% of workload. Now we are managing 9000+ computers through our Meshcentral server. I really appreciate all the team members and contributors behind this sweet project. 
I really like this tiny and powerful tool... This project is beyond my imaginations.

Thanks @Ylianst @krayon007 and all other contributors.

**Comment by jacotec** [comment]
Hey,

I'm Marco and found this great project years too late ;-) ... it flies too much under the radar.
I'm a German guy and in my main job I'm a software (embedded) and hardware engineer focused on industrial automation and CTO of a small company. But my biggest hobby is all that IT stuff ... servers, networking, communication etc. Running some Dell servers at home and one in a data center. I love the idea of Open Source!

We've used Teamviewer in the last years in the company and I've also purchased a NPO license for my sports club to support guys (which will also phase out in a few years) - but their business model (you can't connect to newer versions even for simple remote desktop) was always a pain in my a**. I was waiting for a chance to get rid of this money printing machine for years and MC did it.

Deployed MeshCentral for me and for my company and I'm pretty happy. Still potential for enhancements, but it's an awesome project. As I'm not that much on the programming in JS I hope I can contribute a bit with testing, ideas and reporting issues.

**Comment by PathfinderNetworks** [comment]
Hello all,
My name is Parrish and I'm from southern Michigan.  I'm an IT professional/small business owner and have been in IT for about 30 years now.  Up until 2004 I was the IT Manager for a local power company then went full-time with my own IT services business that same year.  I'm one of the long time users of MeshCentral starting with version 1 back in 2014.  I came to it after LogMeIn started losing their minds and began doing annual price increases of astronomical amounts. So I had to move my customer base from LogMeIn to something else.  Luckily I found MeshCentral and it's been a superb solution ever since.  Especially because we can self host it (which means I don't have to worry about a third party business or server being involved with the remote access to my customers machines).  
I'm definitely not a coder as my specialty is network engineering and hardware (my actual education is in electrical engineering).  So I mostly just suggest things I'd like to see or bugs I find and let Ylian and his team handle the rest. ;-)

**Comment by rallisf1** [comment]
Hi all,
My name is John and I run a small IT business in Athens, Greece. I am currently investigating the opportunity to monitor all our servers and PCs, provide support to our customers and also give domain access to clients who want to use it for Work-From-Home or provide support to their clients, all through a single High-Availability MeshCentral installation. I have a demo installation running atm on which I already customized some of the views, integrated with our ticket system and looking into code signing the agents.

I can definetely provide bug reports and contribute a little to the codebase (mostly front-end stuff).
